addons/Media_Recorder/src/presenter.js

```diff
@@ -3916,10 +3916,15 @@ var BaseRecorder = exports.BaseRecorder = function (_Recorder) {
     _createClass(BaseRecorder, [{
         key: "startRecording",
         value: function startRecording(stream) {
+            var _this2 = this;
```

Comment:
osobiście wolę self zamiast "this2", ale to drobnostka

Reply:
Jest to zmiana wygenerowana z webpacka. Czy po wygenerowaniu niej powinniśmy coś w tym pliku zmieniać ?